src/app/GitUI/CommandsDialogs/FormSubmodules.cs

```diff
@@ -99,12 +100,9 @@ private void Initialize()
             };
             _bw.ProgressChanged += (sender, e) =>
             {
-                lock (_modules)
```

Comment:
The old double lock seem to be there from the start in 2012, should be OK to remove